src/model/transaction/splitBlockInContentState.js

```diff
@@ -46,6 +46,7 @@ function splitBlockInContentState(
     key: keyBelow,
     text: text.slice(offset),
     characterList: chars.slice(offset),
+    data: Object(),
```

Comment:
I can see it both ways. 

For instance, if block data was used to store 'depth' (instead of a dedicated property), I guess the reasonable thing to do would be to keep it. But if data was used to store an anchor location, that would not be ideal. 

Might it be useful to keep it but offer a callback (that pops-up to DraftModifier.splitBlock) to allow manipulating the new block? Or is this overkill?